walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,62 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    
+    private static List<TimeRange> sortTimesByStart(List<TimeRange> timeRanges) {
+      timeRanges.sort(TimeRange.ORDER_BY_START);
+      return timeRanges;
+    }
```

Comment:
You should have your helper functions outside of the query function, and instead make them members of the FindMeetingQuery class.